app/models/course/group_user.rb

```diff
@@ -0,0 +1,17 @@
+class Course::GroupUser < ActiveRecord::Base
+  stampable
+  belongs_to :user, inverse_of: :group_users
+  belongs_to :group, class_name: Course::Group.name, inverse_of: :group_users,
```

Comment:
It's needed, otherwise it will find <code>group_id</code> instead, which does not exists.